### PR TITLE
[93X] several updates for the SiStrip Payload Inspector classes

### PR DESCRIPTION
**Pull request by mmusich**
Summary of the proposed changes:
   * fixed a bug in the logic of the `SiStripApvGainsRatioMaxDeviationWithPreviousIOVTrackerMap` class
   * added back summaries of average gains per barrel layer and endcap disk (stashed at https://github.com/cms-sw/cmssw/pull/19193/commits/d29497908ea5453cb7a24c93ce40743d1923a223) profiting of the newly added class `StandaloneTrackerTopology`
   * added 2D Histograms of gains vs layer (by @jozzez1)
   * added several comparison plots between IOVs and summary by partition for the `SiStripApvGain` object
   * added summary by region for the `SiStripDetVOff`
   * added Payload Inspector for `SiStripBadStrip` object

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
please test

File: CondCore/SiStripPlugins/interface/SiStripPayloadInspectorHelper.h [hunk 1]
```diff
@@ -3,29 +3,223 @@
...
+  /*--------------------------------------------------------------------*/
+  std::pair<float,float> getExtrema(TH1 *h1,TH1 *h2)
+  /*--------------------------------------------------------------------*/
+  {
+    float theMax(-9999.);
+    float theMin(9999.);
+    theMax =  h1->GetMaximum() > h2->GetMaximum() ?  h1->GetMaximum() :  h2->GetMaximum();
+    theMin =  h1->GetMinimum() < h2->GetMaximum() ?  h1->GetMinimum() :  h2->GetMinimum();
+    
+    auto result = std::make_pair(theMin*0.95,theMax*1.05);
```
Comment (davidlange6) [inline CondCore/SiStripPlugins/interface/SiStripPayloadInspectorHelper.h]: hi @mmusich - these are appropriate only for positive max/min values. Is that the only scenario for these functions?

Comment (mmusich) [inline CondCore/SiStripPlugins/interface/SiStripPayloadInspectorHelper.h]: hi @davidlange6, at the moment I am using it only for quantities defined positive, like counts or averages of gains (enforced to be positive), but I do not exclude one might want to recycle this method for quantities than can become negative, so I have taken your comment. Thanks

**Comment by mmusich** [comment]
Hi @smuzaffar the bot is requiring me to apply a patch for code style in a file I didn't even touch: see https://github.com/cms-sw/cmssw/pull/20184#issuecomment-323945615. 
Something should be wrong in the checks. Thanks

**Comment by smuzaffar** [comment]
@mmusich , it must be due to the fact that #20207 was merged and picked up by the test.
I will fix the test to only consider files changed in the PR.

**Comment by davidlange6** [comment]
please test

**Comment by mmusich** [comment]
@ggovi can you sign this again? Thanks!

**Comment by ggovi** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**